packages/liveblocks-devtools/src/devtools/components/Tree.tsx

```diff
@@ -326,6 +367,31 @@ function yLogsColor(node: YLogsTreeNode): string {
   }
 }
 
+export function getYTreeNodeColor(node: YTreeNode): string | undefined {
+  switch (node.type) {
+    case "Y.AbstractType":
+      return "text-brand-500 dark:text-brand-400";
+    case "Y.Map":
+      return "text-red-500 dark:text-red-400";
+    case "Y.Array":
+      return "text-green-500 dark:text-green-500";
+    case "Y.Text":
+    case "Y.XmlText":
+      return "text-blue-500 dark:text-blue-400";
+    case "Y.XmlElement":
+    case "Y.XmlFragment":
+      return "text-orange-500 dark:text-orange-400";
+
+    case "Y.Doc":
+    case "Y.ContentAny":
+    case "Y.ContentBinary":
+    case "Y.ContentEmbed":
+    case "Y.ContentFormat":
+    case "Y.ContentString":
+      return "text-dark-500 dark:text-light-500";
+  }
```

Comment:
We should use the same colors for the same icons, so `Y.XmlFragment` should be the same color as `Y.XmlText` since they both use `<StringIcon />` for example, `Y.ContentString` too, etc. And ideally only `<EllipsisIcon />` should be gray.

Reply:
Good point! Will need to do the same for the dashboard colors